libbeat/scripts/load-dashboards.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Usage examples:
+# env KIBANA_INDEX='.kibana_env1' ./load.sh
+# ./load.sh -url http://test.com:9200
+# ./load.sh -url http://test.com:9200 -user admin:secret
+# ./load.sh -url http://test.com:9200 -index .kibana-test
+
+# The default value of the variable. Initialize your own variables here
+ELASTICSEARCH=http://localhost:9200
+CURL=curl
+KIBANA_INDEX=".kibana"
+KIBANA_PATH?=""
```

Comment:
The load-dashboards.sh script is used to load the dashboard/(s) for a single Beat instead for all of them, right?